transport-native-epoll/src/main/c/io_netty_channel_epoll_Native.c

```diff
@@ -1182,7 +1182,11 @@ JNIEXPORT void JNICALL Java_io_netty_channel_epoll_Native_setKeepAlive(JNIEnv* e
 }
 
 JNIEXPORT void JNICALL Java_io_netty_channel_epoll_Native_setTcpCork(JNIEnv* env, jclass clazz, jint fd, jint optval) {
-    setOption(env, fd, SOL_TCP, TCP_CORK, &optval, sizeof(optval));
+    setOption(env, fd, IPPROTO_TCP, TCP_CORK, &optval, sizeof(optval));
```

Comment:
`SOL_TCP` is an isolated define from `netinet/tcp.h`.  `IPPROTO_TCP` is defined with all other "common" protocols in `netinet/in.h` and is the recommended enumeration to use for TCP [1].  These definitions are both functionally equivalent and resolve to `6` however I believe `IPPROTO_TCP` is more correct to use.  In addition `SOL_TCP` is no longer visible now that we don't include `netinet/tcp.h`.

[1] http://linux.die.net/man/7/tcp